lib/twitter/rest/account_activity.rb

```diff
@@ -0,0 +1,99 @@
+require 'twitter/rest/request'
+require 'twitter/rest/utils'
+require 'twitter/utils'
+
+module Twitter
+  module REST
+    module AccountActivity
+      include Twitter::REST::Utils
+      include Twitter::Utils
+
+      # Registers a webhook URL for all event types. The URL will be validated via CRC request before saving. In case the validation failed, returns comprehensive error message to the requester.
+      # @see https://developer.twitter.com/en/docs/accounts-and-users/subscribe-account-activity/api-reference
+      # @see https://developer.twitter.com/en/docs/accounts-and-users/subscribe-account-activity/api-reference/aaa-premium#post-account-activity-all-env-name-webhooks
+      # @note Create a webhook
+      # @rate_limited Yes
+      # @authentication Requires user context - all consumer and access tokens
+      # @raise [Twitter::Error::Unauthorized] Error raised when supplied user credentials are not valid.
+      # @return [Hash]
+      # @param env_name [String] Environment Name.
+      # @param url [String] Encoded URL for the callback endpoint.
+      def create_webhook(env_name, url)
+        perform_request(:post, "/1.1/account_activity/all/#{env_name}/webhooks.json", url: url)
```

Comment:
The previous version was `perform_request(:json_post, "/1.1/account_activity/all/#{env_name}/webhooks.json?url=#{url}")` but it gives authentication error. It's the same when I use `:post` instead of `:json_post`.

I tried with curl and have the same authentication error when I use a URL argument.